en/docs/how-tos/security/api-authorization/subscription-validation.md

```diff
@@ -1,91 +1,73 @@
     ```
+   
+    For JWT, provide JWT issuer configurations. Follow document on [configure Multiple JWT issuers]({{base_path}}/how-tos/security/api-authentication/secure-apis-using-oauth2.0-access-tokens/secure-apis-using-jwt-self-contained-jwt/#configure-multiple-jwt-issuers).
+    For Opaque / reference token, provide introspection configurations. Follow document on [Secure APIs Using Opaque Tokens]({{base_path}}/how-tos/security/api-authentication/secure-apis-using-oauth2.0-access-tokens/secure-apis-using-opaque-tokens/#configure-api-microgateway-to-validate-oauth2-opaque-tokens).
+    
+    !!! note
+        Older versions of WSO2 API Manager (2.x series and 3.0.0, 3.1.0) do not have the Event Hub capabilities. Therefore, when using Microgateway with those APIM versions, disable the Event Hub and set enableLegacyKM to true. This will use the older key validation service to validate the token and subscriptions for reference/ opaque tokens.
+    
+        ```yml
+        # Key manager configurations
+        [keyManager]
+        # Connection URL of the Key Manager server
+        serverUrl = "https://localhost:9443"
+        # The token endpoint context of the Key Manager server
+        tokenContext = "oauth2"
+        # When Microgateway is used with older APIM versions for subscription validation by using KeyValidation service.
+        enableLegacyMode = true
+        ...
+        [apim.eventHub]
+        # Enable / Disable the feature
+        enable = false
+        ```
+        
+2. Enable subscription validation.
 
-### Enable Subscription Validation
-You can enable or disable subscription validation using the following configuration and it is disabled by default. Add the following to the `<MGW-RUNTIME_HOME>`/conf/micro-gw.conf.
-
-!!! note
-    In order to subscription validation, the ``` [eventHub] ``` must be enabled.
-
-#### Enable Subscription Validation for JWT tokens.
-``` yaml 
-  [[jwtTokenConfig]]
-    issuer = "https://localhost:9443/oauth2/token"
-    audience = "http://org.wso2.apimgt/gateway"
-    certificateAlias = "wso2apim310"
-    # Validate subscribed APIs
-    validateSubscription = true
-```
-
-#### Enable Subscription Validation for Opaque/ Reference tokens.
-``` yaml
-  [security]
-    # Enable/ Disable subscription validation for tokens.
-    validateSubscriptions = true
-```
+    Enable Subscription Validation for JWT tokens by enabling `validateSubscription` under the corresponding JWT issuer configuration.
+    
+    ```toml 
+    [[jwtTokenConfig]]
+        issuer = "https://localhost:9443/oauth2/token"
+        audience = "http://org.wso2.apimgt/gateway"
+        certificateAlias = "wso2apim310"
+        # Validate subscribed APIs
+        validateSubscription = true
+    ```
+        
+    Enable Subscription Validation for Opaque/ Reference tokens by enabling `validateSubscriptions` in security configuration.
```

Comment:
This will enable subscription validation globally. That means for jwt tokens as well. But in the jwt issuer level it can be overridden.